docker-compose.emily.yml

```diff
@@ -0,0 +1,73 @@
+services:
+
+  # Hosts the DynamoDB Tables.
+  dynamodb:
+    image: "amazon/dynamodb-local:latest"
+    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ./data"
+    ports:
+      - "8000:8000"
+    volumes:
+      - "./devenv/dynamodb/data:/home/dynamodblocal/data"
+    networks:
+      - aws-local-vpc
+
+  # Modifies the CDK template and creates DynamoDB Tables if necessary.
+  aws-setup:
+    build: devenv/aws-setup
+    depends_on:
+      dynamodb:
+        condition: service_started
+    volumes:
+      - "./devenv/aws-setup/initialize.py:/initialize.py"
+      - "./emily/cdk/cdk.out:/cdk.out"
+    environment:
+      - AWS_DEFAULT_REGION=us-west-2
+      - AWS_ACCESS_KEY_ID=xxxxxxxxxxxx
+      - AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxx
```

Comment:
I assume these placeholders doesn't do anything. Can we omit them completely?

Reply:
Nope, they need to be set unfortunately

Reply:
Nice :+1: